.github/copilot-instructions.md

```
@@ -0,0 +1,508 @@
+- The current run's JSON (e.g., `current.windows-latest.x64.json`)
+- The resolved baseline JSON (e.g., `CacheBenchmarks.windows-latest.x64.json`)
+- Markdown and HTML reports
+- Test results and coverage
+
+**No baseline is ever overwritten in CI.**
+
+**Debugging/forensics:**
+- You can always download both the current and baseline JSON for any failed job to compare and investigate.
+
+**Example artifact names:**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)